### PR TITLE
LPS-101644 Deprecate `Liferay.Util.disableEsc`

### PR DESCRIPTION
**Pull request by julien**
The goal of this change is to mark the `Liferay.Util.disableEsc`
function as deprecated.

This function is used in a few places:

In the `archived` directory, which we don't want to update, but don't want to
break either
https://github.com/liferay/liferay-portal/blob/1991cde6e2f3ef10689a7b32c819cd91cf9faa55/modules/apps/archived/flash-web/src/main/resources/META-INF/resources/configuration.jsp#L61
https://github.com/liferay/liferay-portal/blob/1991cde6e2f3ef10689a7b32c819cd91cf9faa55/modules/apps/archived/web-proxy-web/src/main/resources/META-INF/resources/configuration.jsp#L56

And in current modules that are still "active"
https://github.com/liferay/liferay-portal/blob/aba1c8126c487579a96eeddd977f1f1ecd91ae3b/modules/apps/iframe/iframe-web/src/main/resources/META-INF/resources/configuration.jsp#L144
https://github.com/liferay/liferay-portal/blob/3750e154e3b73509e0cda8c5f77ebb47a099b4de/portal-web/docroot/html/taglib/ui/input_field/page.jsp#L520
https://github.com/liferay/liferay-portal/blob/3750e154e3b73509e0cda8c5f77ebb47a099b4de/portal-web/docroot/html/taglib/ui/input_field/page.jsp#L529

For now, marking this function as deprecated seems like the safest
option, in the future if we wan't to disable the "escape" key, it might
be worth relying on something else than `window.event.returnValue`, as
it doesn't seem to be supported in Firefox as mentioned here
https://developer.mozilla.org/en-US/docs/Web/API/Event/returnValue

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js [hunk 1]
```diff
@@ -115,6 +115,15 @@ Liferay.Util = Liferay.Util || {};
  */
 Liferay.Util.addParams = addParams;
 
+/**
+ * @deprecated As of Athanasius (7.3.x), with no direct replacement
+ */
+Liferay.Util.disableEsc = () => {
+	if (document.all && window.event.keyCode == 27) {
```
Comment (wincent) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js:122]: `===` here please.

Will be automatically enforced once https://github.com/liferay/eslint-config-liferay/pull/171 gets merged and propagated here, but in the meantime, we shouldn't add any more uses of `==`.

**Comment by wincent** [comment]
Just one little tweak needed please. Going to stop CI in the meantime.

**Comment by wincent** [comment]
ci:stop

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js (hunk 1 above)
Comment (julien) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/global.es.js:122]: I actually thought that would be the case but wasn't 100% sure.

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by wincent** [comment]
ci:forward

**Pull request closed without merging**